packages/types/src/interfaces/extrinsic/v0/ExtrinsicPayloadV0.ts

```diff
@@ -32,9 +32,9 @@ export interface ExtrinsicPayloadValueV0 {
 }
 
 // The base of an extrinsic payload
-export const BasePayloadV0: Record<string, any> = {
+export const BasePayloadV0: Record<string, keyof InterfaceTypes> = {
   method: 'Bytes',
-  doughnut: 'Option<Doughnut>',
+  doughnut: 'Option<doughnut>',
```

Comment:
may be we need 'Option<doughnut>',
which is import from 
import { ChargeTransactionPayment, doughnut, Index } from '../../types';